app/views/content/life-as-a-teacher/change-careers.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Career change to teaching"
+heading: "Change to a career in teaching"
+category: Life as a teacher
+image: "static/images/content/hero-images/physics.jpg"
+description: |-
+   Find out what extra support you can get if you want a career change into teaching. Bring your skills and experience to life in the classroom.
+navigation: 40.25
```

Comment:
Do we need this `navigation` line now - thought this referred to old category page layout (but I might be wrong!)

If not, would be worth having a quick ticket on the board to clear up any other instances in Life as a Teacher (separate PR)

Reply:
We need it as it's used for the order of the pages in the navigation drop down.